### ISSUE TITLE
Unable to install Kitematic on 2014 Macbook Air

### ISSUE THREAD

**Issue by dischord01** [kitematic/kitematic#381]
I have tried several time to install Kitematic on my 2014 Macbook Air, I think the problem is that it is trying to create a VM that is 2GB in size and I only have 4GB of memory. Is there a way to reduce the VM size to 1.5 or less?

```
/Users/ssollar/Downloads/Kitematic (Beta).app/Contents/Resources/app/resources/docker-machine-   0.1.0-kitematic-0.5.10 create -d virtualbox --virtualbox-memory 2048 dev returned non zero exit code stdout:time="2015-04-16T13:25:19-04:00" level="warning" msg="Unable to check for the latest release: Get https://api.github.com/repos/boot2docker/boot2docker/releases: dial tcp 192.30.252.139:443: connection refused" time="2015-04-16T13:25:19-04:00" level="info" msg="Downloading boot2docker.iso to /Users/ssollar/.docker/machine/cache/boot2docker.iso..." time="2015-04-16T13:25:19-04:00" level="error" msg="Error creating machine: Get : unsupported protocol scheme \"\"" time="2015-04-16T13:25:19-04:00" level="warning" msg="You will want to check the provider to make sure the machine and associated resources were properly removed." time="2015-04-16T13:25:19-04:00" level="fatal" msg="Error creating machine" stderr:
```

**Comment by timothyjlaurent** [comment]
add this to /etc/hosts

```
127.0.0.1 localhost
```
This fixed the problem for me.

**Comment by dischord01** [comment]
Here is what I have for /etc/hosts

```
127.0.0.1       localhost
255.255.255.255 broadcasthost
::1             localhost
fe80::1%lo0     localhost
```

Still can not install.

**Comment by jeffdm** [comment]
Fixed in [0.5.18](https://github.com/kitematic/kitematic/releases/tag/v0.5.18)
